meta-resin-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -8,9 +8,10 @@ DOCKER_TIMEOUT=20 # Wait 20 seconds for docker to start
 # Default values
 PARTITION_SIZE=${PARTITION_SIZE:=1024}
 
-# Create a BTRFS disk image
-dd if=/dev/zero of=/export/data_disk.img bs=1M count=$PARTITION_SIZE
-mkfs.btrfs --mixed --metadata=single /export/data_disk.img
+# Create sparse file to hold ext4 resin-data partition
+dd if=/dev/zero of=/export/data_disk.img bs=1M count=0 seek=$PARTITION_SIZE
```

Comment:
We no longer have to go through these hoops to create the partition. It can be done the normal yocto way, i.e start a docker daemon pointed at a folder on the host and then run `mkfs.ext4 -d /path/to/var/lib/docker`